### PR TITLE
Fixes Mush Punch

### LINKED ISSUE #36295: MushPunching doesn't work
Relevant PR: #35858 (can't see it cuz arm is flagged lol!)

you may need to call basic_attack(A,D) in harm_act before returning, dunno.

### PR DESCRIPTION
**Pull request by ghost**
Yeah so while I was working on power kicks I found out that returning TRUE ends the attack correctly and returining FALSE actually makes the person do a basic hit, seriously who made martial arts code man

understandable if i don't get a token for this I put in a broken thing myself
fixes #36295

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
what the actual heck

**Comment by ExcessiveUseOfCobblestone** [comment]
So why isn't putting it on basic_hit not working?

**Comment by Armhulen** [comment]
martial

arts

code

**Comment by Armhulen** [comment]
whatever this is much better than changing basic hit

**Comment by Qbopper** [comment]
I wasn't that much against this PR, but OP appears to be retarded. That earns a solid 👎  from me.

**Comment by Armhulen** [comment]
I wasn't that much against this PR, but OP appears to be a git. That earns a solid 👋🐸 from me

File: code/datums/martial/mushpunch.dm [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-	return FALSE
+	if(!do_after(A, 25, target = D))
+		to_chat(A, "<span class='spider'><b>Your attack was interrupted!</b></span>")
+		return TRUE //martial art code was a mistake
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	atk_verb = pick("punches", "smashes", "ruptures", "cracks")
+	D.visible_message("<span class='danger'>[A] [atk_verb] [D] with inhuman strength, sending [D.p_them()] flying backwards!</span>", \
+					  "<span class='userdanger'>[A] [atk_verb] you with inhuman strength, sending you flying backwards!</span>")
+	D.apply_damage(rand(15,30), BRUTE)
+	playsound(get_turf(D), 'sound/effects/meteorimpact.ogg', 25, 1, -1)
```
Comment (Cyberboss) [inline code/datums/martial/mushpunch.dm]: remove get_turf

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Pull request merged**